packages/esm-ward-app/src/routes.json

```diff
@@ -3,6 +3,9 @@
   "backendDependencies": {
     "webservices.rest": "^2.2.0"
   },
+  "optionalBackendDependencies":{
+     "bedmanagement":"^5.14.0"
+  },
```

Comment:
Not entirely sure what the point of the `optionalBackendDependencies` is.  It doesn't seem to do anything here, and it's unclear to me what it _should_ do?

Reply:
@mseaton It's intended to be a bit similar to `aware_of` modules, basically allowing us to turn certain features on if the backend can support them. Currently, it differs from `backendDependencies` only in that it won't fail if the module _isn't_ installed, but will fail if the wrong version of the module is installed.

Reply:
Basically see https://openmrs.atlassian.net/browse/O3-3266.